apps/dash-manufacture-spc-dashboard/assets/spc-custom-styles.css

```diff
@@ -0,0 +1,794 @@
+  color: white !important;
+}
+
+/*Placeholder*/
+.Select--single > .Select-control .Select-value, .Select-placeholder {
+  /*border: 1px solid darkgray;*/
+  border-radius: 4px;
+  background-color: #242633;
+  cursor: pointer;
+}
+
+
+.Select.is-focused:not(.is-open) > .Select-control {
+  border-color: #91dfd2 !important;
+}
+
+/* Container
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+
+#app-container {
+  background: #161a28;
+  margin: 1rem 2rem;
+  max-width: 100%;
+  width: 100%;
+  height: calc(100vh - 10rem - 1px);
+}
+
+#app-content {
+  background: inherit;
+  padding: 0;
+  width: 100%;
+  max-width: 100%;
+}
+
+#settings-menu {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  justify-content: space-evenly;
+  margin: 4rem 5rem 1rem;
+}
+
+#metric-select-menu {
+  flex: 1 1 0;
+}
+
+#value-setter-menu {
+  flex: 1 1 30%;
+  margin-left: 5rem;
+  display: flex;
+  flex-direction: column;
+}
+
+#status-container {
+  width: 100%;
+  max-width: 100%;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin: 0;
+  padding: 0;
+}
+
+#graphs-container {
+  display: flex;
+  flex-direction: column;
+  flex: 3 1;
+  max-width: calc(75% - 0.8rem);
+  width: 70%;
+}
+
+
+/* Tab 2 Quick stats
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+#quick-stats {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  border-right: #23262E solid 0.8rem;
+  flex: 1 1;
+  padding: 2rem;
+  max-width: 25%;
+}
+
+#operator-led {
+  width: 200px;
+}
+
+#operator-led > div {
+  width: 80%;
+  margin: auto;
+}
+
+#operator-led > div > div {
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+
+#operator-led > div > div svg:nth-child(1) {
+  margin-left: -1rem;
+}
+
+#progress-gauge > div, #progress-gauge > div > div, #progress-gauge > div > div > div {
+  width: 100%;
+  text-align: center;
+}
+
+#progress-gauge svg {
+  width: 80%;
+  height: auto;
+}
+
+#progress-gauge > div > div > div > div {
+  color: white !important;
+}
+
+/*pressure-gauge track*/
+.eUJyCY .track {
+  stroke: #f4d44d !important;
+  stroke-width: 10;
+}
+
+.eUJyCY .track.progress {
+  stroke: #fff3c5 !important;
+}
+
+/* Tab 2 Specs
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+#stop-button > div > button {
+  color: white !important;
+  background-color: #1d202d !important;
+  border: darkgray solid 1px !important;
+  border-radius: 4px;
+  height: 40px;
+  line-height: 40px;
+}
+
+#stop-button > div > button:hover {
+  color: #91dfd2 !important;
+  border-color: white !important;
+  background-color: #1d202d !important;
+}
+
+#card-1 P, #card-2 P {
+  color: darkgray;
+  font-weight: bold;
+  align-self: flex-start;
+}
+
+#card-1 P, #card-2 P {
+  margin: 2rem;
+  font-weight: bold;
+  color: inherit;
+}
```

Comment:
Wondering if you intended to fuse these two:
```
#card-1 P, #card-2 P {
  color: darkgray;
  font-weight: bold;
  align-self: flex-start;
}

#card-1 P, #card-2 P {
  margin: 2rem;
  font-weight: bold;
  color: inherit;
}
```
To one and keep the `darkgray` colour instead of inheriting?
```
#card-1 P, #card-2 P {
  color: darkgray;
  font-weight: bold;
  align-self: flex-start;
  margin: 2rem;
}
```

Reply:
merged.